packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -38,7 +48,7 @@ const StyledButtonSmall = styled(Button)`
   padding: ${({ theme }) => `${theme.spaces[1]} ${theme.spaces[3]}`};
 `;
 
-export const MenuItem = ({ children, onClick, to, isFocused, ...props }) => {
+export const MenuItem = ({ children, onClick, to, isFocused, href, isExternal, ...props }) => {
```

Comment:
Some comments on `SimpleMenu` v1:
- padding/style is currently broken for external `Links` because the padding doesn't take the icon into account, to fix it I propose:
```js
const OptionExternalLink = styled(Link)`
  padding: ${({ theme }) => theme.spaces[2]};
  ...
`;

...

<OptionExternalLink href={href} {...menuItemProps}>
  <Typography>{children}</Typography>
</OptionExternalLink>
```
- sorry if I was unclear about `isExternal` in my last comments, I meant that we don't need `isExternal` in `Link` v1 or `SimpleMenu` v1, since we integrate react-router we only need to and `href` prop `to` know if it is an internal or external link that we want. Could you remove isExternal from `MenuItem` v1 prop + remove it from the condition when rendering?
```js
 else if (href)
    return (
      <OptionExternalLink href={href} {...menuItemProps}>
        <Typography>{children}</Typography>
      </OptionExternalLink>
    );
```
- I'm not sure why you added a condition to properly pass the ref to each items, we need it - without it focus/tabbing won't work
```js
  const menuItemProps = {
    tabIndex: isFocused ? 0 : -1,
    ref: menuItemRef,
    role: 'menuitem',
    ...props,
  };
```